metrics/web/lib/dashboard/domain/entities/metrics/build_number_metrics.dart

```diff
@@ -1,16 +1,16 @@
 import 'package:equatable/equatable.dart';
 
-/// Represents the build number metric entity.
-class BuildNumberMetric extends Equatable {
+/// Represents the build number metrics entity.
+class BuildNumberMetrics extends Equatable {
```

Comment:
```suggestion
/// Represents the build number metric entity.
class BuildNumberMetric extends Equatable {
```
It looks like this is relevant to 1 metric - so let's leave it as it was before.